.github/workflows/main.yml

```
@@ -1,24 +1,25 @@
-# Run My App's tests whenever someone pushes to `main` or creates a PR into `main`
-
 name: CI
-
 on:
-  # Triggers the workflow on push or pull request events but only for the main branch.
   push:
     branches: [ main ]
   pull_request:
     branches: [ main ]
 
-# We're only going to have one job, `test` but you can add this
 jobs:
   test:
     runs-on: macOS-latest
+    
     steps:
-	    # Checkout the code to the working directory
-      - uses: actions/checkout@v2
-      # Install xcbeautify so the logs are human-friendly
-      - name: install xcbeautify
+      - uses: actions/checkout@v2 #checkout working dir
```

```suggestion
      - uses: actions/checkout@v4 #checkout working dir
```
Recommend v4 for increased stability, security, and improved logging.

---

functionally it is the same